server/models/contractor.js

```diff
@@ -0,0 +1,31 @@
+//contractor: skills, jobs completed.  Proposals: contractor
+//name, cost estimate, start date estimate, time frame estimate
+
+const Base = require('./index');
+
+const Contractor = Base.discriminator('Contractor', new mongoose.Schema({
+    skills: {
+        
+        required: true,
+    },
```

Comment:
Similar comment as on how to handle Jobs on Customer applies here. I'm leaning to using references since proposals will be associated with a contractor and a job.